src/main/java/chalmers/pimp/model/ModelMemento.java

```diff
@@ -11,13 +12,16 @@
 public final class ModelMemento {
 
   private final CanvasMemento canvasMemento;
+  private final ColorModelMemento colorModelMemento;
 
   /**
-   * @param canvasMemento a memento object for a canvas instance.
+   * @param canvasMemento     a memento object for a canvas instance.
+   * @param colorModelMemento         the selected color.
    * @throws NullPointerException if any references are {@code null}.
    */
-  ModelMemento(CanvasMemento canvasMemento) {
+  ModelMemento(CanvasMemento canvasMemento, ColorModelMemento colorModelMemento) {
     this.canvasMemento = Objects.requireNonNull(canvasMemento);
+    this.colorModelMemento = Objects.requireNonNull(colorModelMemento);
   }
```

Comment:
Are the javadoc parameters aligned properly?

Reply:
No it is not. Having some problem with my formatter. Fixed it though